cli/cli_set_technology.go

```diff
@@ -19,9 +19,12 @@ const (
 	SetTechnologyUsageText     = "Sets the technology"
 	SetTechnologyArgsUsageText = `<technology>`
 	SetTechnologyDescription   = `Use this command to set the technology.
-Supported values for <technology>: OPENVPN or NORDLYNX.
+Supported values for <technology>: %s.
 
 Example: 'nordvpn set technology OPENVPN'`
+
+	SupportedValuesNoQuench = "OPENVPN or NORDLYNX"
+	SupportedValuesQuench   = "OPENVPN, NORDLYNX or QUENCH"
```

Comment:
Name needs clarifications

Reply:
This needs to be `NordWhisper`. Might be good to have all places replaced with this, if not at least all the user facing texts need to be updated.